DuckDuckGoTests/LargeOmniBarStateTessts.swift

```diff
@@ -0,0 +1,338 @@
+//
```

Comment:
The filename is LargeOmniBarStateTe**ss**ts.swift. Did you use Xcode function to refactor? :P

Reply:
Thanks, will fix